### PR TITLE
note in the docs for 2->3 migration

### PR DESCRIPTION
**Pull request by mircealungu**

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Please note that the changes before version 1.10.0 have not been documented.
 
+v3.0.0
+----------
+Changed
+
+- Tracking also status codes
+
```
Comment (mircealungu) [inline docs/changelog.rst]: @bogdanp05 @FlyingBird95 @thejager -- anything else we should be adding here?

Comment (FlyingBird95) [reply docs/changelog.rst]: Changes:

- Display times as numbers to make them sortable 7ebfef6
- Add leading slash to blueprint paths b370407
- Added status codes with corresponding views a8e84d9

Comment (mircealungu) [reply docs/changelog.rst]: Cool. Updated file correspondingly.

**Comment by FlyingBird95** [comment]
Really  nice @mircealungu !

But don't forget to update the version to 3.0.3 ;)

Patrick

**Comment by mircealungu** [comment]
@FlyingBird95 - version updated.

**Pull request merged**